CHANGELOG

```diff
@@ -5,6 +5,32 @@ Legend:
          !! bug fixed
 
 =========================================
+0.8.3-Belitto       2018mmdd
```

Comment:
Now when we know when to relase we can update the date which is then the commit to add the tag for 0.8.3.
I think the only thing left is to update the website.